2019-2020_season.html

```diff
@@ -29,17 +30,18 @@
   <nav>
     <li>
       <div class="dropdown">
-        <a href="index.html">Mission</a>
+        <a href="#">Mission</a>
         <div class="dropdown-content">
           <ul>
-            <li><a href="mission-faq.html">FAQ</a></li>
+            <li><a href="index.html#mission">Mission</a></li>
+            <li><a href="mission-faq.html">Mission FAQ</a></li>
```

Comment:
making the dropdowns all have `#` hrefs and then proper links on click means you can use them on mobile